modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/rest-openapi.yaml

```diff
@@ -211,6 +211,11 @@ components:
                 content:
                     type: string
                     x-json-string: true
+                creator:
+                    $ref: "#/components/schemas/Creator"
+                    description:
+                        The person who created the workflow
```

Comment:
Change the description to:  `The workflow's creator`.

Reply:
@vngoliveira, This matches the existing descriptions of the Rest APIs from the portal.

Reply:
Thanks @marcelabc!